src/tools/compiletest/src/header.rs

```diff
@@ -877,7 +879,13 @@ pub fn make_test_description<R: Read>(
             return;
         }
         ignore = match config.parse_cfg_name_directive(ln, "ignore") {
-            ParsedNameDirective::Match => true,
+            ParsedNameDirective::Match => {
+                #[cfg(not(bootstrap))]
+                {
+                    ignore_message = config.parse_meta_name_value_directive(ln, "ignore");
+                }
```

Comment:
Let's not try to parse an ignore_message at all in compiletest, it's not likely to work well with the current syntax on the ground.

Reply:
Okay, the parser is removed